run-all-examples.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+cargo build --examples
```

Comment:
This is fine, although it shouldn't be necessary, as `cargo run --example` will build the example first.